config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Is it necessary to keep this? It is all commented out.

Reply:
Yeah; these files were generated along with the rails upgrade command. I'll remove it so we can add it if we require it.

Reply:
They are useful to see what defaults are available and they should be removed **gradually** by enabling and testing. I think, they should be kept for the next upgrade story (changing the defaults).